src/test/resources/alma-fix/990174080470206441.json

```diff
@@ -0,0 +1,297 @@
+{
```

Comment:
Not sure whether introducing of this (and other) documents is superflous as the old test files already cover the case.

Reply:
This is the only test file that does not introduce any new info. But should we keep it so that I do not spent to much time rebuilding the branch?